lib/active_resource/connection.rb

```diff
@@ -295,5 +296,14 @@ def legitimize_auth_type(auth_type)
         auth_type = auth_type.to_sym
         auth_type.in?([:basic, :digest]) ? auth_type : :basic
       end
+
+      def get_request_params(params_string)
```

Comment:
Maybe pass the format and the raw body and let the log subscriber deal with the parsing? This way there is no overhead on this feature unless the users are subscribing to this event